lib/resources/r4/patient.yaml

```diff
@@ -81,15 +76,13 @@ fields:
 
 - name: communication
   required: 'No'
-  cardinality: 0..*
-  type: BackboneElement
+  type: List of BackboneElement
   description: Language which may be used to communicate with the patient about his or her health.
   action: terminology
   children:
 
   - name: language
     required: 'No'
```

Comment:
double check here and identifier

Reply:
[I changed location](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-07c686ec4248977e084ba5a602c761ceL85), but I think identifier is correct.